src/main/java/heronarts/lx/midi/surface/APC40Mk2.java

```diff
@@ -698,26 +717,8 @@ private void register() {
     for (LXAbstractChannel channel : this.lx.engine.mixer.channels) {
       registerChannel(channel);
     }
-    this.lx.engine.mixer.addListener(new LXMixerEngine.Listener() {
-
-      @Override
-      public void channelRemoved(LXMixerEngine mixer, LXAbstractChannel channel) {
-        unregisterChannel(channel);
-        sendChannels();
-      }
-
-      @Override
-      public void channelMoved(LXMixerEngine mixer, LXAbstractChannel channel) {
-        sendChannels();
-      }
-
-      @Override
-      public void channelAdded(LXMixerEngine mixer, LXAbstractChannel channel) {
-        sendChannels();
-        registerChannel(channel);
-      }
-    });
 
```

Comment:
If you can just toss a this.mixerEngineListener there - I know it's not necessary but I make a habit of doing this. It makes it very clear to the reader that the variable is a member of this class... not of a containing outer class, or a function parameter, etc.